GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Engines.cfg

```diff
@@ -1047,7 +1047,39 @@
 	}
 }
 
-@PART[sepMotor1|ROSmallSpinMotor|SnubOtron|sepMotorLarge]:FOR[RealismOverhaul]
+//  ==================================================
+//  Radial Separation motor (small).
+//  ==================================================
+
++PART[sepMotor1]:AFTER[RealismOverhaul]
+{
+	@name = sepMotorSmall
+	@title = Radial Separation Motor (Small)
+	@description = Small solid motor use to help separate one stage from another or perform ullage. Best used with others.
+
+	%rescaleFactor = 0.5
+	@mass = 0.008
+
+	@MODULE[ModuleEngines*]
+	{
+		@maxThrust *= 0.18
+	}
+
+	@MODULE[ModuleFuelTanks]
+	{
+		@volume *= 0.25
+	}
+
+	@MODULE[ModuleEngineConfigs]
+	{
+		@CONFIG[SolidFuel]
+		{
+			@maxThrust *= 0.18
+		}
+	}
+}
+
+@PART[sepMotor1|ROSmallSpinMotor|SnubOtron|sepMotorLarge|sepMotorSmall]:FOR[RealismOverhaul]
 {
```

Comment:
If you duplicated in :AFTER[RealismOverhaul] then it wasn't available to run against in FOR[RealismOverhaul].
Which isn't to say your change is wrong, the original patch was.  But remove sepMotorLarge and sepMotorSmall from here, because they don't exist yet.

Reply:
should they be added somewhere else?

Reply:
https://github.com/KSP-RO/RealismOverhaul/pull/2454/commits/a8ebe6e95f3177993a8c629348f33143f84ea0c5